README.md

```diff
@@ -365,3 +365,42 @@ is not initialized yet. Also you may want to stop using Vagrant and you want to
 wipe out all Vagrant control files and resources from your workstation. For those purposes you
 should consider using the `.ci/vagrant-cleaner.sh` script; run `.ci/vagrant-cleaner.sh -h` for
 further information.
+
+
+## Using the kata-local-ci libvirt-based script
+
+Another way to setup a local CI is tu use the `cmd/kata-local-ci` script. This
+script uses `libvirt` in order to setup a virtual machine in which the CI
+environment is created. This VM is then snapshotted immediately after setup, so
+that the same environment can be restored quickly.
+
+Compared to the `vagrant` approach, the main benefit is that this setup can
```

Comment:
Again, you could drop the backticks as vagrant is a dictionary word.

Reply:
Used Vagrant the first time, but `vagrant` for the second instance, because I'm really referring to the command.